Extension/lib/filter/rules/css-filter.js

```diff
@@ -407,6 +407,8 @@ CssFilter.prototype = {
 	 * Firefox only.
 	 *
 	 * Builds CSS stylesheet to be registered browser-wide.
+	 * This stylesheet contains all rules, but ignoring exceptions. Exception rules gonna be applied

```

Comment:
Browser-wide stylesheet contains all element-hiding rules except "extended CSS" rules.

NOTE: elemHideIntercepter.js decides what to do with the element matching selectors from this stylesheet. For instance, elemHideIntercepter checks rule domain permissions.